scripts/modules/book_class/find_class.py

```diff
@@ -29,6 +29,7 @@ def find_class(self, today, confs, classes):
         for c in classes[class_name]:
           testing_time = date.strptime(c['t'], '%H:%M')
           if (testing_time - testing_now).total_seconds() < 9 * 3600 and testing_now < testing_time and start_time < testing_time and end_time > testing_time:
+            c['id'] = c['id'].replace('aula', '', 1)
```

Comment:
You could either return the result of the `replace()` function, or in the parser trim the *aula*.